manifests/d/Drumstick/dmidiplayer/1.6.0/Drumstick.dmidiplayer.installer.yaml

```
@@ -0,0 +1,28 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS0.5-1-19041-1682
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: Drumstick.dmidiplayer
+PackageVersion: 1.6.0
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: /S
+UpgradeBehavior: install
+FileExtensions:
+- kar
+- mid
+- midi
+- rmi
+- wrk
+ReleaseDate: 2022-05-17
+Installers:
+- Architecture: x64
+  InstallerUrl: http://downloads.sourceforge.net/project/dmidiplayer/v1.6.0/dmidiplayer-1.6.0-win-x64-setup.exe
```

```suggestion
  InstallerUrl: https://sourceforge.net/projects/dmidiplayer/files/v1.6.0/dmidiplayer-1.6.0-win-x64-setup.exe/download
```
for consistency with other sourceforge packages.